server/src/utils/validations/common/isEmail.js

```diff
@@ -0,0 +1,13 @@
+import addErrorIfNotValid from '../addErrorIfNotValid';
+
+const isEmail = (email) => {
+  const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
did you write this regex? 
do you understand this regex ?

Reply:
Picked it up from [this website](https://emailregex.com/), tested it on regexr.com. I understand it. Covers all email cases I could find.